opentech/apply/review/views.py

```diff
@@ -181,6 +270,12 @@ def get_context_data(self, **kwargs):
 
         for i, review in enumerate(self.object_list):
             review_data['title']['answers'].append('<a href="{}">{}</a>'.format(review.get_absolute_url(), review.author))
+            if review.opinions:
```

Comment:
do you need the if statement? Perhaps select related the authors on opinions again.

if there are no opinions could the template return nothing/minimal markup? 

You can use an `empty` template tag to handle this nicely.